src/components/Classrooms.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import fetch from 'isomorphic-fetch';
+
+export default class Classrooms extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      data: [{
+        attributes: { },
+        id: '',
+        relationship: { },
+        type: ''
+      }]
+    };
+  }
+  // Fetch classrooms from education-api and update state
+  componentDidMount() {
+    this.fetchClassrooms = fetch(this.props.source, {
+      method: 'GET',
+      mode: 'cors',
+      headers: new Headers({
+          'Authorization': 'Bearer ' + '64aa60e488a555e2ef8b7f9d1621ec7e430dcb8b8baa0d00b8460eca5660d188',
+          'Content-Type': 'application/json'
+      })
+    })
+    .then(function(res) {
```

Comment:
You've used this pattern in `actions.js`, along with fat arrows to save binding context:

```
promise
    .then(response => response.json())
    .then(json => {
        console.log('componentDidMountSTATE: ', this.state);
        this.setState({data: json.data})
        console.log('componentDidMountSTATE: ', this.state)
    });